go/bump-vendored-golang/task.yml

```diff
@@ -19,6 +19,7 @@ run:
 params:
   BLOBSTORE_ACCESS_KEY_ID: # ID of the blobstore access key for the bosh release
   BLOBSTORE_SECRET_ACCESS_KEY: # Secret of the blobstore access key for the bosh release
+  GCS_JSON_KEY: # GCP service account key to access the GCS bucket
```

Comment:
It would be kind to note here that GCS_JSON_KEY and BLOBSTORE_ACCESS_KEY_ID/BLOBSTORE_ACCESS_KEY_ID are mutually exclusive.